Content/Console/build.fsx

```diff
@@ -161,12 +226,40 @@ let clean _ =
     ]
     |> Seq.iter Shell.rm
 
+let updateChangelog ctx =
+    let description, unreleasedChanges =
+        match changelog.Unreleased with
+        | None -> None, []
+        | Some u -> u.Description, u.Changes
+    let verStr = ctx |> getVersionNumber "RELEASE_VERSION"
+    let newVersion = SemVer.parse verStr
+    if changelog.Entries |> List.exists (fun entry -> entry.SemVer = newVersion) then
+        let entry = changelog.Entries |> List.find (fun entry -> entry.SemVer = newVersion)
+        Trace.traceErrorfn "Version %s already exists in %s, released on %s" verStr changelogFilename (if entry.Date.IsSome then entry.Date.Value.ToString("yyyy-MM-dd") else "(no date specified)")
+        failwith "Can't release with a duplicate version number"
```

Comment:
How do you feel about:

```fsharp
    changelog.Entries
    |> List.tryFind(fun entry -> entry.SemVer = newVersion)
    |> Option.iter(fun entry ->
        Trace.traceErrorfn "Version %s already exists in %s, released on %A" verStr changelogFilename (if entry.Date.IsSome then entry.Date.Value.ToString("yyyy-MM-dd") else "(no date specified)")
        failwith "Can't release with a duplicate version number"
    )
```

and this style for the similar section below.

Reply:
Sure. That's a bit more functional, and only goes through the list once instead of twice. I'll do that.

Reply:
Done in dbc1a55.